scim-spec/scim-spec-schema/src/test/java/org/apache/directory/scim/spec/schema/SchemaTest.java

```diff
@@ -71,7 +71,6 @@ public void setUp() {
       "schemas/urn:ietf:params:scim:schemas:core:2.0:ServiceProviderConfig.json",
       "schemas/urn:ietf:params:scim:schemas:extension:enterprise:2.0:User.json"
   })
-  @Disabled
```

Comment:
As the commit message says, the filenames for the schemas are not Windows friendly, so this test was failing.

I cannot even rename them without using a different OS.  I'll file a new low-priority issue, since I'm not sure what the maintainers would want to do with them...